modules/end-user-guide/pages/git-credential-store.adoc

```diff
@@ -1,25 +1,48 @@
-:navtitle: Git credential store
+:navtitle: Creating a {orch-name} Secret for a Git credentials store
 :keywords: user-guide, configuring, user, secrets
 :page-aliases: 
 
-[id="mounting-a-git-credential-store-into-a-workspace-container_{context}"]
-= Mounting a Git credentials store into a workspace container
+[id="mounting-a-git-credential-store-into-workspace-containers_{context}"]
+= Creating a {orch-name} Secret for a Git credentials store
 
-This section describes how to mount Git credentials store as secret from the user's {orch-namespace} into the file in single-workspace or multiple-workspace containers of {prod-short}.
+This section describes how to create a Secret containing a Git credentials store in the user's {orch-namespace}. The Git credentials store specified in the Secret is mounted as a file in the {devworkspace}'s containers.
 
 .Prerequisites
 
 * A running instance of {prod-short}.
+* You have CLI or GUI access to the {platforms-name} cluster of your organization's {prod-short} instance.
 
 .Procedure
-. Prepare Git credential file in the link:https://git-scm.com/docs/git-credential-store#_storage_format[Storage format].
-. Encode content of the file to the base64 format.
-. Create a new {platforms-name} secret in the {platforms-name} {orch-namespace} where a {prod-short} workspace will be created.
+. Prepare a Git credential file in the link:https://git-scm.com/docs/git-credential-store#_storage_format[Storage format].
```

Comment:
```suggestion
. In your home directory, locate and open your `.git-credentials` file if you already have it. Alternatively, if you do not have this file, then save a new `.git-credentials` file, using the following link:https://git-scm.com/docs/git-credential-store#_storage_format[format] per file line:
[subs="+quotes,+attributes,+macros"]
----
https://username:token@__<git_server_hostname>__
----
+
[TIP]
====
CLI users can use the `git config` command to create a `.git-credentials` file:
. `$ git config --global credential.helper store`
. `$ git push`
. Verify that the `.git-credentials` file has been created in the home directory.
====

.A line in a `.git-credentials` file
====
`https://jane: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@github.com`
====
```

Reply:
I think this tip is something I'd like to leave out for simplicity since its already explained in https://git-scm.com/docs/git-credential-store. Also, for this tip, it's not clear to where and what the user is pushing. My new commit also adds https://git-scm.com/docs/git-credential-store as an `Additional resource` at the bottom of the page:

![image](https://user-images.githubusercontent.com/83611742/163057651-d507c38e-777a-411f-8696-8c0090d29ee8.png)

Reply:
@dkwon17, I suggested the `[TIP]` based on some third-party how-to article online when I was trying to figure out with Google what https://git-scm.com/docs/git-credential-store meant. Have a second look, it doesn't seem to be there. This simplifies things (like looking up tokens) for users who have local Git repo clones.

Reply:
David, you can resolve this thread.
I already discarded my suggestion in https://github.com/eclipse-che/che-docs/pull/2275#discussion_r849615575.